test/runtime/math_test.cpp

```diff
@@ -71,6 +71,20 @@ TEST_CASE("'fma()' on quantity changes the value and the dimension accordingly",
   REQUIRE(fma(isq::speed(10.0 * m / s), isq::time(2.0 * s), isq::height(42.0 * m)) == isq::length(62.0 * m));
 }
 
+TEST_CASE("fmod functions", "[math][fmod]")
+{
+  SECTION("fmod should work on the same quantities")
+  {
+    REQUIRE(fmod(4. * isq::length[km], 3. * isq::length[km]) == 1. * isq::length[km]);
+    REQUIRE(fmod(-9. * isq::length[km], 3. * isq::length[km]) == -0. * isq::length[km]);
+  }
+  SECTION("fmod should work with different units of the same dimension")
```

Comment:
Could you please also provide a similar test for an integral representation type?